source/js/components/people/person.jsx

```diff
@@ -13,9 +13,26 @@ export default class Person extends React.Component {
   }
 
   flip() {
+    if(this.state.flipped) {
+      this.setState({
+        'flippableStyle': {
+          height: this.refs.bioContent.clientHeight
+        }});
+    } else {
+      this.setState({
+        'flippableStyle': {
+          height: this.refs.quoteContent.clientHeight
+        }});
+    }
     this.setState({'flipped': !this.state.flipped});
   }
 
+  componentDidMount() {
+    this.refs.quoteContent && this.setState({'flippableStyle': {
```

Comment:
I wouldn't use this pattern outside JSX as it's slightly hacky despite their endorsement in that context.

Just use a standard `if` here.